### PR TITLE
Merge the latest modification branch to master

### PR DESCRIPTION
**Pull request by bilalelhoudaigui**
This branch contains the latest changes.

### CODE REVIEW

**Comment by cpommier** [comment]
I am on it

File: etl/transform/elasticsearch.py [hunk 1]
```diff
@@ -13,173 +18,152 @@
...
+    for document_tuple in document_tuples:
+        document_count += 1
+        yield dict(document_tuple)
+    logger.debug(f"Generated {document_count} documents.")
 
 
-def generate_elasticsearch_document(options):
-    document_type, document_transform, data_id, data_index = options
-    document = data_index[data_id]
+def generate_elasticsearch_document2(options):
```
Comment (cpommier) [inline etl/transform/elasticsearch.py]: generate_elasticsearch_documents seems to be deprecated and replaced with generate_elasticsearch_document2
COnsider removing generate_elasticsearch_documents and rename generate_elasticsearch_document2 to generate_elasticsearch_documents

File: etl/transform/elasticsearch.py [hunk 2]
```diff
@@ -13,173 +18,152 @@
...
+    document = json.loads(data_json)
     if document_transform:
-        resolved = remove_empty(resolve(document_transform, document, data_index))
+        resolved = remove_empty(resolve(document_transform, document, uri_data_index))
         document.update(resolved)
     return document_type, document
 
 
-def generate_elasticsearch_documents(restricted_documents, document_configs_by_entity, data_index, pool, logger):
+def generate_elasticsearch_documents2(restricted_documents, document_configs_by_entity,
```
Comment (cpommier) [inline etl/transform/elasticsearch.py]: Idem as above, keep that one and remove the deprecated one

**Comment by cpommier** [comment]
@bilalelhoudaigui , did you had the opportunity to rerun the whole ETL pipeline up to an interface loading?

**Comment by bilalelhoudaigui** [comment]
> @bilalelhoudaigui , did you had the opportunity to rerun the whole ETL pipeline up to an interface loading?

Yes @cpommier, the ETL went smoothly without any issues.

**Comment by cpommier** [comment]
Perfect lets merge

**Pull request merged**